create_description/launch/create_2.launch

```diff
@@ -3,4 +3,6 @@
   <node name="robot_state_publisher" pkg="robot_state_publisher" exec="robot_state_publisher" output="screen">
     <param name="robot_description" type="str" value="$(command '$(find-pkg-prefix xacro)/bin/xacro $(find-pkg-share create_description)/urdf/create_2.urdf.xacro')" />
   </node>
+  <!--Bumper to Point Cloud-->
+  <node pkg="rviz_scripts" exec="bumperToPointCloud" name="bumperToPointCloud" output="screen" />
```

Comment:
This creates a runtime dependency on `rviz_scripts`. I'd rather not introduce a dependency for visualization to this description package.
```suggestion
```